phoenix-hive-base/pom.xml

```diff
@@ -195,101 +180,75 @@
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>3.0.0</version>
                 <executions>
                     <execution>
-                        <id>HBaseManagedTimeTests</id>
+                        <id>add-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
                         <configuration>
-                            <encoding>UTF-8</encoding>
-                            <forkCount>1</forkCount>
-                            <runOrder>alphabetical</runOrder>
-                            <reuseForks>false</reuseForks>
-                            <argLine>-enableassertions -Xmx2000m -XX:MaxPermSize=256m
-                                -Djava.security.egd=file:/dev/./urandom
-                                "-Djava.library.path=${hadoop.library.path}${path.separator}${java.library.path}"
-                                -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=./target/
-                                -Dorg.apache.hadoop.hbase.shaded.io.netty.packagePrefix=org.apache.hadoop.hbase.shaded.
-                            </argLine>
-                            <redirectTestOutputToFile>${test.output.tofile}
-                            </redirectTestOutputToFile>
-                            <testSourceDirectory>${basedir}/src/it/java</testSourceDirectory>
-                            <groups>org.apache.phoenix.end2end.HBaseManagedTimeTest</groups>
-                            <shutdown>kill</shutdown>
-                            <useSystemClassLoader>false</useSystemClassLoader>
+                            <sources>
+                                <source>${project.parent.basedir}/src/main/java</source>
+                            </sources>
                         </configuration>
+                    </execution>
+                    <execution>
+                        <id>add-test-source</id>
+                        <phase>generate-sources</phase>
                         <goals>
-                            <goal>integration-test</goal>
-                            <goal>verify</goal>
+                            <goal>add-test-source</goal>
                         </goals>
+                        <configuration>
+                            <sources>
+                                <source>${project.parent.basedir}/src/it/java</source>
+                                <source>${project.basedir}/src/it/java</source>
+                                <source>${project.parent.basedir}/src/test/java</source>
+                            </sources>
+                        </configuration>
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
I think that in this case we simply want to depend on -base, instead of copying stuff from it.